Assets/Scripts/Gamestate/MatchController.cs

```diff
@@ -161,6 +162,28 @@ public void StartNextRound()
                 ai.TrackedPlayers = players.Select(player => player.playerManager)
                     .Where(player => player != ai).ToList());
 
+        if (NetworkManager.singleton.isNetworkActive)
+        {
+            StartCoroutine(WaitForClientsAndInitialize());
+            return;
+        }
+
+        MusicTrackManager.Singleton.SwitchTo(MusicType.Battle);
```

Comment:
Please put this init code into its own method to avoid duplicating all these lines :))))))